app/views/users/index.html.erb

```diff
@@ -6,6 +6,7 @@
     <th>User Role</th>
   </thead>
   <tbody>
+    <% users = User.includes(:role) %>
```

Comment:
why we are using `@users = User.all` in `app/controllers/users_controller.rb` if we are doing a query inside view!?

Reply:
Updated